codespell_lib/data/dictionary.txt

```diff
@@ -3685,16 +3890,24 @@ anaylzer->analyzer
 anaylzers->analyzers
 anaylzes->analyzes
 anaylzing->analyzing
+anaysis->analysis
 anaytic->analytic
 anaytical->analytical
 anaytically->analytically
 anaytics->analytics
 anbd->and
+anc->and
```

Comment:
Could be the [ANC](https://www.oed.com/search/dictionary/?q=anc) (African National Congress). Do you think this should be moved to _rare_?

Reply:
Let's move it to rare.